clang/test/SemaHLSL/BuiltIns/WaveGetLaneIndex-errors.hlsl

```diff
@@ -0,0 +1,6 @@
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only -disable-llvm-passes -verify -verify-ignore-unexpected
```

Comment:
Are you getting unexpected diagnostics? If so, I'd like to understand them. If not, I don't think `-verify-ignore-unexpected` is needed. If anything, we should limit it to the type of diagnostic that is being produced e.g. `-verify-ignore-unexpected=note`

Reply:
The flag was not needed, so have removed it.